TAScheduler/views_methods.py

```diff
@@ -404,6 +405,37 @@ def courseTAAsmgt(self, active_ta, active_course):  # new remove sectionTAAsmgt
             raise RuntimeError("Instructor is already assigned to max number of course permitted")
         TAToCourse.objects.create(ta=active_ta.database, course=active_course.database)
 
+    def sectionTAAsmgt(self, active_ta, active_section):
```

Comment:
I think we usually want our assignment methods to take in "objs" not DBObjs - I see you're comparing active_ta to "TA" (not TAObj) and then you're checking active_section to LabObj

I would recommend comparing them to the class objects

Reply:
The way I made LabObj and LecObj work is they take the database, but I can change them to take TAObj if you want?